templates/zerver/documentation_main.html

```diff
@@ -1,13 +1,13 @@
 {% extends "zerver/portico-help.html" %}
 
-{# Zulip User Documentation. #}
+{# Zulip User and API Documentation. #}
 
 {% block portico_content %}
 <div class="app help terms-page inline-block">
     <div class="sidebar slide">
         <div class="content">
             <h1><a href="/help/" class="no-underline">Index</a></h1>
```

Comment:
this href should be `/api` for the API page.

Reply:
Ah - sorry about that.